build.gradle

```diff
@@ -256,6 +259,11 @@ dependencies {
             }
         }
 
+        // Platform integration with Windows and macOS
+        implementation 'net.java.dev.jna:jna-platform:5.9.0'
```

Comment:
`jna-platform` module has  a native libraries for macOS and windows.
It may be necessary to sign "*.dylib" for macOS...

Reply:
It is signed and no need to do anything.